pkg/controllers/machine_controller.go

```diff
@@ -40,6 +42,7 @@ const (
 	rootFSAlias                     = "ua-rootfs"
 	libvirtDomainXMLIgnitionKeyName = "opt/com.coreos/config"
 	networkInterfaceAliasPrefix     = "ua-networkinterface-"
+	defaultRetryReconcileTime       = 15 * time.Second
```

Comment:
Do we need this reconciletime?

Reply:
```text
5
10
20
40
80
160
320
640
1280 (1.28s)
2560
5120
10240
20480
40960
81920
163840
327680
655360
```
@lukasfrank as you can see, we currently start with 5ms timeout between reconcile loops. So if we will return error for reconcilation, it will call another reconcile loops under 1seconds what is quick for shutdown process because machine need minimal seconds for graceful shutdown. So sourav has problem with this first 10 reconciliation  loops.